frameworks/helloworld/tests/test_dns.py

```diff
@@ -0,0 +1,32 @@
+import dcos
+import pytest
+import shakedown
+
+import sdk_install as install
+import sdk_plan as plan
+import sdk_spin as spin
+
+from tests.config import (
+    PACKAGE_NAME
+)
+
+
+def setup_module(module):
+    install.uninstall(PACKAGE_NAME)
+    options = {
+        "service": {
+            "spec_file": "examples/dns.yml"
+        }
+    }
+
+    # this yml has 2 hello's + 0 world's:
```

Comment:
Cargo-culted from another test, but I figured it was there to indicate why the total number of tasks expected was 2. Will remove.